wlauto/common/android/workload.py

```diff
@@ -481,83 +482,19 @@ def teardown(self, context):
         if self.uninstall_apk:
             self.device.uninstall(self.package)
 
-
 AndroidBenchmark = ApkWorkload  # backward compatibility
 
-
-class ReventWorkload(Workload):
-    # pylint: disable=attribute-defined-outside-init
-
+class ReventWorkload(ReventWorkloadMaster):
+    #Future work can be done for this workload and its subclasses to add support for features detailed below
     def __init__(self, device, _call_super=True, **kwargs):
-        if _call_super:
-            Workload.__init__(self, device, **kwargs)
-        devpath = self.device.path
-        self.on_device_revent_binary = devpath.join(self.device.binaries_directory, 'revent')
-        self.setup_timeout = kwargs.get('setup_timeout', None)
-        self.run_timeout = kwargs.get('run_timeout', None)
-        self.revent_setup_file = None
-        self.revent_run_file = None
-        self.on_device_setup_revent = None
-        self.on_device_run_revent = None
-        self.statedefs_dir = None
-
+        super(ReventWorkload, self).__init__(device, _call_super, **kwargs)
         if self.check_states:
             state_detector.check_match_state_dependencies()
+        self.idle = False #Idle mode is not currently implemented for Android
+        self.no_close = True #Currently support for .close files is not implemented for Android workloads
+        self.default_naming = True #Currently only default-named revent files generated via wa record are implemented for Android workloads
```

Comment:
Its to ensure that the Android version of the Workload is essentially the same as it was before since the Idle mode and .close file features as well as non-default names haven't been done for Android yet, so for now I have just forced the options.